stable/mpi-operator/templates/mpi-operator-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.deployment.create }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "mpi-operator.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "mpi-operator.name" . }}
+    chart: {{ template "mpi-operator.chart" . }}
+    release: {{ .Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mpi-operator
+  template:
+    metadata:
+      labels:
+        app: mpi-operator
+    spec:
+      serviceAccountName: mpi-operator
```

Comment:
suggestion: service account name should be overridable

Reply:
Making both serviceaccount and clusterrole overrideable (since clusterrole is created separately this will come in handy